bool refresh
---
bool success
string message
string[] device_names
string[] host_api_names
int32[] device_indices
uint32[] max_input_channels
uint32[] default_sample_rates
string[] device_ids
